examples/run/dp2.x-lammps-ABACUS-lcao/fcc-al/run_param.json

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
If possible, these two files can be added to `check_examples` as well:

https://github.com/deepmodeling/dpgen/blob/369b9355d451d0e84d3deb7432970bf4d59bedb7/tests/test_check_examples.py#L33-L35

Then we do not need to check it manually.

Reply:
@pxlxingliang, it looks that the variant name in #845 is incorrect, making the tests fail.

Reply:
> @pxlxingliang, it looks that the variant name in #845 is incorrect, making the tests fail.

We plane to modify the fp_style to be "abacus", then @hongriTianqi will push the modification.